recipes/morningstar-data/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "morningstar-data" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: morningstar-data 
+  version: 1.3.0 
+
+source:
+  url: https://files.pythonhosted.org/packages/cd/dd/b90be5d468e9808a69cdea7fee16cd99a5da5eb1e29f5479954b0de93754/morningstar_data-1.3.0.tar.gz
```

Comment:
Please use a different url where you can template the version. Pick the pattern from any other Python package.